config.ru

```diff
@@ -1,17 +1,17 @@
 require_relative 'plugin.rb'
 require_relative 'app/logging.rb'
 
+$stdout.sync = $stderr.sync = true
+
 if ENV['LOG_LEVEL']
   log_level = ENV['LOG_LEVEL'].to_i
 else
   log_level = Logger::INFO
 end
-Logging.initialize_logger(STDOUT, log_level)
+Logging.initialize_logger(STDERR, log_level)
```

Comment:
A bit 50:50, becuase `thin` appears to log to STDOUT... but IMO logging should always go to STDERR in general, otherwise it corrupts the output of any CLI app trying to use that code while writing data to STDOUT.

Fixing to use STDERR here for consistency with `kontena-etcd` logging